src/hotspot/share/utilities/resourceHash.hpp

```diff
@@ -76,14 +80,13 @@ class ResourceHashtableBase : public STORAGE {
     return ptr;
   }
 
-  Node const** lookup_node(unsigned hash, K const& key) const {
-    return const_cast<Node const**>(
-        const_cast<ResourceHashtableBase*>(this)->lookup_node(hash, key));
```

Comment:
I would keep this the const version of lookup_node(), since that seems more correct. I would use it in contains() instead of using get(), which does too much.